ci-scripts/yocto2.groovy

```diff
@@ -247,37 +210,64 @@ void buildManifest(String variantName, String imageName, String layerToReplace="
         boolean smokeTests = getBoolEnvVar("SMOKE_TEST", false)
         boolean bitbakeTests = getBoolEnvVar("BITBAKE_TEST", false)
         boolean yoctoCompatTest = getBoolEnvVar("YOCTO_COMPATIBILITY_TEST", false)
-        setupBitbake(yoctoDir, templateConf, doArchiveCache, smokeTests, analyzeImage)
-        setupCache(yoctoDir, yoctoCacheURL)
+        stage("Setup bitbake and cache") {
+            setupBitbake(yoctoDir, templateConf, doArchiveCache, smokeTests, analyzeImage)
+            setupCache(yoctoDir, yoctoCacheURL)
+        }
 
         // Build the images
         try {
+            stage("Bitbake ${imageName} for ${variantName}") {
+                buildImage(yoctoDir, imageName, variantName)
+            }
+
             boolean buildUpdate = variantName.startsWith("rpi") || variantName.startsWith("intel") || variantName.startsWith("arp")
-            buildImageAndSDK(yoctoDir, imageName, variantName, buildUpdate)
+            stage("Bitbake Update ${imageName} for ${variantName}") {
+                when (buildUpdate) {
+                    sh "/workspace/cookbook/yocto/build-images.sh ${yoctoDir} ${imageName}-update"
+                }
+            }
+
+            boolean buildSDK = getBoolEnvVar("BUILD_SDK", false)
+            stage("Build SDK ${imageName}") {
+                when (buildSDK) {
+                    sh "/workspace/cookbook/yocto/build-sdk.sh ${yoctoDir} ${imageName}"
+                }
+            }
+
             if (yoctoCompatTest) {
                 runYoctoCheckLayer(yoctoDir)
             }
             if (smokeTests) {
                 runSmokeTests(yoctoDir, imageName)
-                if(bitbakeTests) {
+            }
```

Comment:
I have cleaned up it yesterday as on stand-up we decided to not show it as we don't use smoke tests. Same for bitbake tests, but noone knows  what is it, so we left it at the moment to investigate

Reply:
Some misunderstanding here, I think. We want to show the stage if it is enabled. Right?

Reply:
The stage should always be on place. We can just skip it and it will be empty.

Reply:
My understanding is that the stage always needs to be visible, but will be empty in case the stage is not executed. So stages for smoke test and bitbake tests should always be visible in the view, right?